app/Resources/views/base.html.twig

```diff
@@ -88,6 +88,9 @@
 
         {% block footer %}
             <hr>
+            <ul id="footerLinks">
```

Comment:
Looking at https://github.com/SURFnet/Stepup-RA/blob/6cd272c55f0e3ec962324cd66c6529a373720a4d/src/Surfnet/StepupRa/RaBundle/Resources/views/SecondFactor/search.html.twig#L61-L61 I figured we were doing camelCase for ids, snake for classes?